setCwd.js

```diff
@@ -4,9 +4,9 @@ const { promisify } = require('util');
 const promiseExec = promisify(exec);
 
 const setCwd = async ({ dispatch, action, tab }) => {
-  const newCwd = await promiseExec(
+  const { stdout } = await promiseExec(
     `lsof -p ${tab.pid} | grep cwd | tr -s ' ' | cut -d ' ' -f9-`);
-  const cwd = newCwd.trim();
+  const cwd = stdout.trim();
```

Comment:
```suggestion
  // Since node v8, return type of a promisified exec has changed: https://github.com/nodejs/node/commit/fe5ca3ff27 
  const cwd = typeof newCwd === 'string' ? newCwd.trim() : newCwd.stdout.trim();
```